WIP 3.0 Classes/Warlock.md

```diff
@@ -304,7 +314,7 @@ You start with the following equipment, in addition to the equipment granted by
 - any simple weapon
 - *(a)* a component pouch or *(b)* an arcane focus
 - *(a)* a scholar's pack or *(b)* an explorer's pack
-- two daggers and a belt pouch
+- a dagger and a belt pouch
```

Comment:
I imagine why the warlock got two of these is because the D&D 5E warlock also gets two as part of its kit. And the reason that class in turn gets two is definitely with throwing them in mind.

So not sure if I see a reason why drop it from 2 to just 1? Small comment but still 😛

Reply:
I can't remember why I did, but I'd imagine my thinking is that two daggers are weapons while one dagger is a tool, like an army knife or athame. Felt more appropriate?